src/clientdlg.cpp

```diff
@@ -1094,6 +1098,18 @@ void CClientDlg::OnTimerSigMet()
     // show current level
     lbrInputLevelL->SetValue ( pClient->GetLevelForMeterdBLeft() );
     lbrInputLevelR->SetValue ( pClient->GetLevelForMeterdBRight() );
+
+    if ( bDetectFeedback && pClient->GetLevelForMeterdBLeft() > 7.5 )
+    {
+        // mute locally and mute channel
+        chbLocalMute->setCheckState ( Qt::Checked );
+        MainMixerBoard->MuteMyChannel();
+
+        QMessageBox::warning ( this, APP_NAME, tr ( "Audio feedback detected.\n"
```

Comment:
Open discussion here: Shouldn't the MUTED label be used instead of a real warning message?

Reply:
I suspect that unexperienced user will never discover the *cause* of muting without an explicit warning message.

Reply:
Probably true. I'll leave this open for other thoughts. By testing it I'd say that the current behavior is better.